### ISSUE TITLE
Opening user commands file points to wrong path

### ISSUE THREAD

**Issue by jakebathman** [twolfson/sublime-edit-command-palette#2]
When using the command palette to open `Preferences: Commands - User`, the wrong file path is used.

Expected file to open:

    C:\Users\<username>\AppData\Roaming\Sublime Text 3\Packages\User\Default.sublime-commands

Actual file opened (actually a new file, since this file doesn't really exist) uses a recently used root path and appends the above path, such as:

    C:\Users\<username>\Projects\projectfolder\another\C:\Users\jbathman\AppData\Roaming\Sublime Text 3\Packages\User

Here's a screenshot of what it looks like on Sublime 3:

![](http://i.imgur.com/sSA0kwS.png)

**Comment by twolfson** [comment]
Well that's strange. Let me see if I can reproduce.

**Comment by twolfson** [comment]
I have reproduced and debugging the source of the issue.

**Comment by twolfson** [comment]
Strange... `open_file` seems to want a UNIX path:

- `window.run_command('open_file', {'file': 'C:\\Users\\IEUser\\AppData\\Roaming\\Sublime Text 3\\Packages\\User\\Default.sublime-commands'})`
    - Opens `{{cwd}}\C:\Users\IEUser\AppData\Roaming\Sublime Text 3\Packages\User\Default.sublime-commands`
- `window.run_command('open_file', {'file': '/C/Users/IEUser/AppData/Roaming/Sublime Text 3/Packages/User/Default.sublime-commands'})`
    - Opens `C:\Users\IEUser\AppData\Roaming\Sublime Text 3\Packages\User\Default.sublime-commands` as desired

**Comment by twolfson** [comment]
Alright, I have solved it. We can use a non-absolute path for `open_file` but will need the long path for providing default content. I am going to write a patch and try it out.

```python
window.run_command('open_file', {'file': 'Packages/User/Default.sublime-commands'})
```

**Comment by twolfson** [comment]
Oh, damn it. Never mind, it's opening relative to `cwd` =/

**Comment by twolfson** [comment]
Finally got it with:

```python
window.run_command('open_file', {'file': '${packages}/User/Default.sublime-commands'})
```

**Comment by twolfson** [comment]
This has been patched/released in `2.1.0`. Thanks for the bug report!
